5-Accessibility/accessible-form.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="augusticor" />
+    <title>Oscar Bosigas Form</title>
+</head>
+
+<body>
+
+    <h1>Form</h1>
+
+    <div>
+        <form>
+            <fieldset> 
+                <legend> Please enter your contacts details </legend>
+                <label for="name">Your name (required):</label>
+                <input required type="text" id="tname" name="name">
```

Comment:
Please include `aria-required="true"` input's attribute.
```suggestion
                <input required type="text" id="tname" name="name" aria-required="true">
```